### DISCUSSION TITLE
Support '/q/' & '/a/' URLs

### DISCUSSION THREAD

**Discussion by KaKi87** [httpjamesm/AnonymousOverflow#130] [Ideas]
e.g. :

- https://askubuntu.com/q/251206
- https://askubuntu.com/a/389209

Thanks

**Comment by httpjamesm** [comment]
This should be an issue - not a discussion

**Reply by KaKi87** [reply]
I was initially going to create an issue, but didn't find a *feature request* option, so I browsed to discussions and saw there was an *ideas* option, which usually means that the maintainer wants people to submit request there instead of in issues. Sorry if I was wrong here, but then what should I have done ? Thanks

**Reply by httpjamesm** [reply]
In general on GitHub, issues are for all feature requests and bug reports. Discussions are meant for more open ended discussion, like vague ideas on a restructure, requests for feedback, etc.

**Reply by KaKi87** [reply]
Yes but the only issue options are *Bug report*, *New Instance* and *Report a security vulnerability* :sweat_smile:

**Reply by httpjamesm** [reply]
You can click the "Don't see your issue here? Open a blank issue." button

**Reply by KaKi87** [reply]
Maintainers usually don't want people to do that.
